<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseTracker</class>
 <widget class="QDialog" name="ExpenseTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1549</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover:!pressed
{
  	border: 4px solid white;
	background-color: red;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>1431</width>
     <height>951</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 59);</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page_TrangChinh">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>390</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#ffffff;&quot;&gt;Page 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_ChiTieu">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>290</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#ffffff;&quot;&gt;Page 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_ThuNhap">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>290</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#ffffff;&quot;&gt;Page 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_TaiKhoan">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>380</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#ffffff;&quot;&gt;Page 4&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_ThongKe">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>360</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; color:#ffffff;&quot;&gt;Page 5&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_NguoiDung">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#ffffff;&quot;&gt;Chủ quản lý tài khoản&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Images/Pictures/User_Avatar_.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_HoVaTen_page6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Họ và tên</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TenDangNhap_page6_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tên đăng nhập:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Email_page6_1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CongViec_page6_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>390</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Công việc:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TenDangNhap_page6_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>250</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tên đăng nhập</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Email_page6_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CongViec_page6_2">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>390</y>
       <width>191</width>
       <height>69</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Công việc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ThayMaPin">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>550</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
color: rgb(0, 0, 0);
background-color: rgb(255, 49, 52);
font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Thay mã pin</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>951</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 39, 63);</string>
   </property>
   <widget class="QPushButton" name="pushButton_TrangChinh">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1px;
color: rgb(0, 0, 0);
background-color: rgb(255, 111, 113);
font: 80 15pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>TRANG CHỦ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ThuNhap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Thu nhập</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NguoiDung">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Người dùng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TaiKhoan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Tài khoản</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ThongKe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Thống kê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChiTieu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Chi tiêu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fullscreen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>917</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 127);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>FullScreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_restoredown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>879</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 127);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Restore Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minimize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 127);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Minimize</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
